.github/workflows/ci.yml

```diff
@@ -8,7 +8,29 @@ on:
   workflow_dispatch:
 
 jobs:
-  run:
+  lint:
```

Comment:
Another solution would be to move the lint workflow to its own workflow file. It should allow to trigger the job independently since GitHub doesn't allow this (at least not yet).